### PR TITLE
feat: add clearer interactive states for dropdown

### PR DESCRIPTION
**Pull request by gregogun**
Updates hover and active states to make interactions between dropdown items clearer for users with low-vision.

### CODE REVIEW

File: sigle/src/stitches.config.ts [hunk 1]
```diff
@@ -69,10 +69,14 @@ const greenDark = {
 };
 
 const red = {
```
Comment (gregogun) [inline sigle/src/stitches.config.ts:71]: I've just added the colors being used in this pr to the ds - just wondering if this is fine or if it's better practice to add all of the green and red colors as they are now in the ds on figma?

File: sigle/src/ui/DropdownMenu.ts [hunk 2]
```diff
@@ -5,6 +5,7 @@ export const DropdownMenu = DropdownMenuPrimitive.Root;
 export const DropdownMenuTrigger = DropdownMenuPrimitive.Trigger;
 export const DropdownMenuSeparator = styled(DropdownMenuPrimitive.Separator, {
   height: 1,
+  mx: '-$3',
```
Comment (gregogun) [inline sigle/src/ui/DropdownMenu.ts:8]: make separator full width despite parent padding

File: sigle/src/stitches.config.ts (hunk 1 above)
Comment (pradel) [reply sigle/src/stitches.config.ts:71]: Should be good like this 👍

**Review by pradel — APPROVED** [review]
Nice 👌

**Pull request merged**